crates/jstz_node/src/lib.rs

```
@@ -199,8 +196,7 @@ pub async fn run(options: RunOptions) -> Result<()> {
     let listener = TcpListener::bind(format!("{addr}:{port}")).await?;
     axum::serve(listener, router).await?;
 
-    cancellation_token.cancel();
-    tail_file_handle.await.unwrap()?;
+    log_service_handle.shutdown().await?;
```

I don't think we need to propagate the error to the parent like we do for storage sync as long as we print out the error - wdyt about implementing `AsyncDrop` instead of calling the shutdown ? @huancheng-trili

---

This log service will be obsolete soon anyway so I don't think this needs to be further refined.

---

Ok I won't furthur refactor - since this is done can we still merge this branch?